neuralhydrology/datasetzoo/camelsind.py

```diff
@@ -0,0 +1,345 @@
+import os
+import pandas as pd
+from pathlib import Path
+from tqdm import tqdm
+from functools import reduce
+from typing import List, Dict, Union
+import xarray
+from neuralhydrology.datasetzoo.basedataset import BaseDataset
+from neuralhydrology.utils.config import Config
```

Comment:
The inputs are not correctly sorted and in general it seems like the file was not formatted. Would be great if you could run yapf over this file. If you have the default neuralhydrology environment installed, you can either modify you IDE to autoformat on save (e.g. possible in visual studio vode) and by pointing to the .style.yapf file. Or you run yapf on the file manually from the terminal with something like

```
/path/to/yapf -i /path/to/camelsind.py --style /path/to/.style.yapf
```

the path to the yapf binary can be found by `which yapf` if you have the conda environment activated. The .style.yapf file is in the root directory of this repository.